File: Java/Beginner/1035/Main.java
import java.util.Scanner;

class Main {

    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);

        int A = scan.nextInt(), B = scan.nextInt(), C = scan.nextInt(), D = scan.nextInt();

        if((B > C && D > A) && (C+D > A+B) && (C > 0 && D > 0) && (A % 2 == 0)) {
            System.out.println("Valores aceitos");
        } else System.out.println("Valores nao aceitos");

        scan.close();
    }

}

File: C/Beginner/1035/main.c
#include <stdio.h>

int main(void) {

    int a, b, c ,d;
    scanf("%d%d%d%d", &a, &b, &c, &d);

    if((a % 2 == 0) && (c > 0 && d > 0) && (c + d > a +b) && (b > c && d > a)) printf("Valores aceitos\n");
    else printf("Valores nao aceitos\n");

    return 0;
}